src/graph/graphmetacontext.h

```diff
@@ -0,0 +1,20 @@
+/*
+* Copyright 2018-2020 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#pragma once
+
+#include "graphcontext.h"
+
+typedef struct {
+	GraphContext *gc;
+	char *meta_key_name;
```

Comment:
We should be able to delete this member if we make the changes suggested in graphencodecontext.h

Reply:
done